ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -468,20 +471,21 @@ export default class ConfirmTransactionBase extends Component {
                         ])}
                       </p>
                       <p>{t('transactionDetailGasTooltipExplanation')}</p>
-                      <p>
+                      {/* <p>
                         <a
                           href="https://community.metamask.io/t/what-is-gas-why-do-transactions-take-so-long/3172"
                           target="_blank"
                           rel="noopener noreferrer"
                         >
                           {t('transactionDetailGasTooltipConversion')}
                         </a>
-                      </p>
+                      </p> */}
                     </>
                   }
                   position="top"
                 >
-                  <i className="fa fa-info-circle" />
+                  {/* <i className="fa fa-info-circle" /> */}
+                  <HelpOutLineIcon />
```

Comment:
```suggestion
                  <HelpOutlineIcon />
```

Reply:
fixed in 0fd7dc7f22cf45741ed4b85432f2dd1a2ea4ae83

Reply:
- Verified